src/index.html

```diff
@@ -13,6 +13,57 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--yellow">
+
+      <div class="stars__row">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars__row">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars__row">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
```

Comment:
Add an empty line between sibling blocks of HTML to add some 'air' and simplify reading. There should be an empty line between the closing </div> of one .stars__row and the opening <div> of the next .stars__row.